erizo/src/erizo/media/MediaProcessor.cpp

```diff
@@ -509,7 +509,7 @@ int OutputProcessor::packageVideo(unsigned char* inBuff, int buffSize, unsigned
         rtpHeader.setTimestamp(av_rescale(pts, 90000, 1000));
     }
     rtpHeader.setSSRC(55543);
-    rtpHeader.setPayloadType(96);
+    rtpHeader.setPayloadType(100);
```

Comment:
This won't work in general since the PT of VP8 is defined in  `rtp_media_config.js`.
The proper solution here would be to pass the mappings in the `ExternalInput` constructor, just like we do for `ExternalOutput` and handle it properly.